appveyor.yml

```diff
@@ -135,6 +135,9 @@ install:
       cd $env:APPVEYOR_BUILD_FOLDER\gdal\swig\python
       exec { python setup.py build }
       exec { python setup.py install }
+      cd $env:APPVEYOR_BUILD_FOLDER\gdal\swig\csharp
+      exec { nmake /f makefile.vc interface}
+      exec { nmake /f makefile.vc all $env:WIN64_ARG}
```

Comment:
environment variable WIN64_ARG defines WIN64 which affect csc option for architecture.